src/details/ArborX_DetailsKokkosExtArithmeticTraits.hpp

```diff
@@ -12,13 +12,35 @@
 #ifndef ARBORX_DETAILS_KOKKOS_EXT_ARITHMETIC_TRAITS_HPP
 #define ARBORX_DETAILS_KOKKOS_EXT_ARITHMETIC_TRAITS_HPP
 
+#include <Kokkos_Macros.hpp>
+#if KOKKOS_VERSION >= 30599
```

Comment:
Does this essentially mean `3.6.0+`?

Reply:
No it means dev version post 3.5 release